src/components/Section/Identification/IdentificationConnector.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import {
+  updateApplication,
+  reportErrors,
+} from '../../../actions/ApplicationActions'
+
+const connectIdentificationSection = (Component, { section, subsection, store, storeKey }) => {
+  class ConnectedIdentificationSection extends React.Component {
+    constructor(props) {
+      super(props)
+  
+      this.section = section
+      this.subsection = subsection
+      this.store = store
+
+      this.handleError = this.handleError.bind(this)
+      this.handleUpdate = this.handleUpdate.bind(this)
+    }
+
+    handleError(value, arr) {
+      const action = reportErrors(this.section, this.subsection, arr)
+      this.props.dispatch(action)
+      return arr
+    }
+  
+    handleUpdate(field, values) {
+      this.props.dispatch(updateApplication(this.store, field, values))
+    }
+
+    render () {
+      return (
+        <Component
+          onUpdate={this.handleUpdate}
+          onError={this.handleError}
+          {...this.props} />
+      )
+    }
+  }
+
+  ConnectedIdentificationSection.propTypes = {
+    Comments: PropTypes.object,
+    update: PropTypes.func,
+    validator: PropTypes.func,
+    dispatch: PropTypes.func, // Passed in via connect (below)
+  }
+
+  const mapStateToProps = (state) => {
+    const app = state.application || {}
+    const identification = app.Identification || {}
+    const errors = app.Errors || {}
+    const completed = app.Completed || {}
+
+    switch (storeKey) {
+      case 'ApplicantName':
+        return { ...identification.ApplicantName } || {}
```

Comment:
Temporary verbosity in order to get parity with how these props are currently passed into each section, can be cleaned up later.